timescaledb/how-to-guides/continuous-aggregates/create-a-continuous-aggregate.md

```diff
@@ -142,14 +142,31 @@ queries to run efficiently.
 
 </procedure>
 
+## Create a continuous aggregate on JOINed tables
+
+In TimescaleDB 2.9 and later, with PostgreSQL 13 or later, you can create a
+continuous aggregate on tables that are connected with a `JOIN`. For example:
```

Comment:
```suggestion
continuous aggregate from a query that uses a `JOIN`. For example:
```

Is this better? The underlying table itself is not permanently connected with a `JOIN`, it's the query (and now the continuous aggregate) that is, so maybe the wording is a bit less confusing? Will bow to the expertise of someone who knows SQL better though